web/modules/custom/joinup_core/joinup_core.drush.inc

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file
+ * Drush commands related to joinup.
+ */
+
+/**
+ * Implements hook_drush_command().
+ */
+function joinup_core_drush_command() {
+  $items = [];
+
+  $items['joinup-status-check'] = [
```

Comment:
Yeah I am aware of that 😐 I discussed it with sander and we agreed on this approach. 

The difference with the rq command is that the rq is a reporting command. This means that the items with severity level: error, do not result as a failure in the terminal which makes it harder to handle. The goal was to have a command that actually fails in case of errors.